.circleci/config.yml

```diff
@@ -445,7 +445,7 @@ workflows:
       - e2e-cypress-tests-master:
           filters:
             branches:
-              only: master
+              only: /^v.*/
```

Comment:
Temporarily setting this to make this PR's tests run with webapp master. Maybe this should be set this anyway?

Reply:
One reason that I don't want to run webapp `master` against all branches is that I don't want to break PRs accidently..

Reply:
Maybe I'm missing something here. This change makes it so that the e2e tests are only run against `v*` branches, I.e. release branches of this repo. Why should it be this way?